File: Downloads/TechTestJava-master/TechTestJava-master/anycompany/src/main/java/za.co.anycompany/model/Customer.java
package za.co.anycompany.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder(toBuilder = true)
public class Customer {
    private int customerId;
    private String name;
    private String country;
    private Date dateOfBirth;

    private List<Order> orders;

    public int getCustomerId() {
        return customerId;
    }
}

File: AnyCompanyEntertainment/AnyCompanyEntertainmentDotNet/Models/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AnyCompanyEntertainmentDotNet.Models
{
    public class Customer
    {
        public Customer()
        {
            this.Orders = new List<Order>();
        }
        public int Id { get; set; }
        public string Name { get; set; }
        public string? country { get; set; }
        public DateTime dateOfBirth { get; set; }
        public List<Order> Orders { get; set; }
    }

}
